Makefile

```diff
@@ -9,6 +9,8 @@ $(FX2LIBDIR)/.git: .gitmodules
 
 # FIXME: Add check_int2jit from hdmi2usb/Makefile
 
+docs: export PROJECT_NUMBER:=$(shell git rev-parse HEAD; git diff-index --quiet HEAD || echo "(with uncommitted changes)")
```

Comment:
You probably want to replace `git rev-parse HEAD; git diff-index --quiet HEAD || echo "(with uncommitted changes)` with `git describe --always --dirty --long`